[Do not review] Test & Debug

xush6528 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32330 [Do not review] Test & Debug**
* #31893 [WIP] Delete all user forks tracked in RRefContext before shutting down

Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

xush6528 committed:
[Do not review] Test & Debug



Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 4a599b26:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=4a599b26486d2f55d0167d4e6baecbc5920074d8).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4286565) pytorch_linux_xenial_py2_7_9_test (1/1)
**Step:** "Set Up CI Environment After attach_workspace" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87488068) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87488068))
<details>
<summary>
<code>Failed to create or replace /initrd.img: No such file or directory at /usr/bin/linux-update-symlinks line 72.</code>
</summary>

```
Setting up libipc-run-perl (0.94-1) ... 
Setting up libnvidia-container1:amd64 (1.0.5-1) ... 
Setting up libnvidia-container-tools (1.0.5-1) ... 
Setting up libtk8.6:amd64 (8.6.5-1) ... 
Setting up linux-firmware (1.157.22) ... 
update-initramfs: Generating /boot/initrd.img-4.15.0-1027-gcp 
W: mdadm: /etc/mdadm/mdadm.conf defines no arrays. 
Setting up linux-modules-4.4.0-171-generic (4.4.0-171.200) ... 
Setting up linux-image-4.4.0-171-generic (4.4.0-171.200) ... 
I: /vmlinuz is now a symlink to boot/vmlinuz-4.4.0-171-generic 
Failed to create or replace /initrd.img: No such file or directory at /usr/bin/linux-update-symlinks line 72. 
dpkg: error processing package linux-image-4.4.0-171-generic (--configure): 
 subprocess installed post-installation script returned error exit status 2 
dpkg: dependency problems prevent configuration of linux-modules-extra-4.4.0-171-generic: 
 linux-modules-extra-4.4.0-171-generic depends on linux-image-4.4.0-171-generic | linux-image-unsigned-4.4.0-171-generic; however: 
  Package linux-image-4.4.0-171-generic is not configured yet. 
  Package linux-image-unsigned-4.4.0-171-generic is not installed. 
 
dpkg: error processing package linux-modules-extra-4.4.0-171-generic (--configure): 
 dependency problems - leaving unconfigured 
Setting up intel-microcode (3.20191115.1ubuntu0.16.04.2) ... 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]

xush6528 committed:
Update on "[Do not review] Test & Debug"


Differential Revision: [D19302763](https://our.internmc.facebook.com/intern/diff/D19302763/)

[ghstack-poisoned]
